RestfulDropApi.cs

```diff
@@ -40,14 +40,10 @@ public RestfulDropApi(string pat)
         /// <param name="blobAPIVersion">The API version to use for the blob API.</param>
         /// <param name="relativeRoot">The root path relative to the drop to retrieve.</param>
         /// <returns>The manifest details.</returns>         
-        public async Task<ISet<VstsFile>> GetVstsManifest(Uri manifestUri, string blobAPIVersion, 
+        public async Task<IList<VstsFile>> GetVstsManifest(Uri manifestUri, string blobAPIVersion, 
                                                                  string relativeRoot)
         {
-            // dotnet core doesn't currently handle vsts redirects well. Poking both teams about it
```

Comment:
I think they did take my bug here :)

Reply:
yeah, seems we don't have to worry about the redirect thing anymore